core-bundle/src/Resources/contao/helper/exception.php

```diff
@@ -8,13 +8,15 @@
  * @license LGPL-3.0-or-later
  */
 
-/**
- * Class UnusedArgumentsException
- *
- * @author Leo Feyer <https://github.com/leofeyer>
- */
-class UnusedArgumentsException extends Exception
+// Register alias in the global namespace for backwards compatibility
+class_exists(\Contao\UnusedArgumentsException::class);
+
+// Let composer find the deprecated class for autoload backwards compatibility
+if (!class_exists('UnusedArgumentsException', false))
```

Comment:
How can this condition ever be `true`? The `class_exists()` call above will register the global alias.

Reply:
Never, but we need composer to register this file for the class map of `UnusedArgumentsException`

Reply:
We could try if we can use `if (false)` instead?

Reply:
That's not necessary IMHO.